### DISCUSSION TITLE
(moved to issue) Chroma background in browser?

### DISCUSSION THREAD

**Discussion by kmyhre** [TuomoKu/SPX-GC#26] [Q&A]
Hi!

Congrats on an excellent project!

I see there is a file `ASSETS/media/images/background/chroma.png`, but I'm not recognising a way to render the website with it (or just some user-defined color) as a background. Is there a fairly easy way to work around this, or is there a plan to enable chroma backgrounds in the UI?

**Comment by kmyhre** [comment]
Changing `background-color` in `spx_layout.css` (either in `body` or in `.SPXWindow`) or uncommenting `background-image` and redirecting it to `../../../../media/images/bg/chroma.png` apparently works – until you need to have more than one object on screen. This results in the other objects appearing behind the background and thus becoming invisible unless the chroma background is transparent – which would arguably defeat the purpose.

**Reply by TuomoKu** [reply]
Hey, there is a (now) an free ImageLayer -template (https://www.spxgc.com/store/ImageLayer-p291331877) which can easily handle images, this is what the chroma.png is there for.

Hope this helps.

**Reply by TuomoKu** [reply]
Starting from v. 1.0.15 there is now an option in Project Settings to add a static background image.

This can be used to add a chroma color or a custom frame border or the like.
